vllm/attention/backends/flashinfer.py

```diff
@@ -676,6 +677,13 @@ def forward(
                 k_scale,
                 v_scale,
             )
+            # The FlashInfer api requires data to be in fp8_e4m3 or fp8_e5m2
+            # to process the cache in fp8
+            if (self.kv_cache_dtype == 'fp8'
+                    or self.kv_cache_dtype == 'fp8_e4m3'
+                    or self.kv_cache_dtype == 'fp8_e5m2'):
```

Comment:
```suggestion
            if self.kv_cache_dtype in ("fp8", "fp8_e4m3", "fp8_e5m2"):
```

Reply:
This seems like it could lead to a user using "fp8_e5m2" cache dtype while getting k_scale and v_scale from a model calibrated for e4m3. Maybe we should force the scales to 1 for that case? This may not be a serious issue since the range is strictly larger for e5m2

Reply:
When I test with `fp8_e5m2`, the model asserts elsewhere. So, if a user specifies `fp8_e5m2`, they wouldn't be able to use this path.